hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
```

Comment:
You forgot to merge HW1 into main before branching off.

Reply:
Yes , I need to merge hw1 to main. I was not sure about it and so did not merge it before.